<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyWidget</class>
 <widget class="QWidget" name="MyWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>637</width>
    <height>666</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>铁路查询</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>124</x>
     <y>260</y>
     <width>78</width>
     <height>29</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;Agency FB&quot;;
font: 14pt &quot;幼圆&quot;;</string>
   </property>
   <item>
    <property name="text">
     <string>北京</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>西安</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>郑州</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>徐州</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>成都</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>广州</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>上海</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>44</x>
     <y>260</y>
     <width>72</width>
     <height>23</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;Agency FB&quot;;
font: 14pt &quot;幼圆&quot;;</string>
   </property>
   <property name="text">
    <string>出发地</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>44</x>
     <y>330</y>
     <width>72</width>
     <height>25</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;Agency FB&quot;;
font: 14pt &quot;幼圆&quot;;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#000000;&quot;&gt;到达地&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_2">
   <property name="geometry">
    <rect>
     <x>124</x>
     <y>330</y>
     <width>78</width>
     <height>29</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;Agency FB&quot;;

font: 14pt &quot;幼圆&quot;;</string>
   </property>
   <item>
    <property name="text">
     <string>北京</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>西安</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>郑州</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>徐州</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>成都</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>广州</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>上海</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>202</x>
     <y>330</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">
border-image: url(:/res/SearchButton2.png);
border-image: url(:/res/SearchButton3.png);
border-image: url(:/res/searchbutton4.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>312</x>
     <y>195</y>
     <width>291</width>
     <height>251</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">

border-image: url(:/res/railway1.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_beijing">
   <property name="geometry">
    <rect>
     <x>442</x>
     <y>175</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;幼圆&quot;;</string>
   </property>
   <property name="text">
    <string>北京</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>40</y>
     <width>241</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/res/tietle1.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>490</y>
     <width>571</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>11</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;幼圆&quot;;

color: rgb(68, 114, 196);
color: rgb(0, 0, 0);
</string>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::North</enum>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <property name="documentMode">
    <bool>false</bool>
   </property>
   <property name="tabsClosable">
    <bool>false</bool>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="tabBarAutoHide">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="tab">
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <attribute name="title">
     <string>距离最短</string>
    </attribute>
    <widget class="QTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>611</width>
       <height>131</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;幼圆&quot;;
color: rgb(0, 0, 0);
border-image: url(:/res/background3.png);
background-image: url(:/res/background3.png);
font: 9pt &quot;幼圆&quot;;</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>时间最短</string>
    </attribute>
    <widget class="QTextEdit" name="textEdit_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>611</width>
       <height>131</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;幼圆&quot;;
color: rgb(0, 0, 0);
border-image: url(:/res/background3.png);
background-image: url(:/res/background3.png);
font: 9pt &quot;幼圆&quot;;</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>费用最少</string>
    </attribute>
    <widget class="QTextEdit" name="textEdit_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>611</width>
       <height>131</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;幼圆&quot;;
color: rgb(0, 0, 0);
border-image: url(:/res/background3.png);
background-image: url(:/res/background3.png);
font: 9pt &quot;幼圆&quot;;</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>中转最少</string>
    </attribute>
    <widget class="QTextEdit" name="textEdit_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>611</width>
       <height>131</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;幼圆&quot;;
color: rgb(0, 0, 0);
border-image: url(:/res/background3.png);
background-image: url(:/res/background3.png);
font: 9pt &quot;幼圆&quot;;</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label_chengdu">
   <property name="geometry">
    <rect>
     <x>282</x>
     <y>405</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;幼圆&quot;;</string>
   </property>
   <property name="text">
    <string>成都</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_xian">
   <property name="geometry">
    <rect>
     <x>312</x>
     <y>285</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;幼圆&quot;;</string>
   </property>
   <property name="text">
    <string>西安</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_zhengzhou">
   <property name="geometry">
    <rect>
     <x>412</x>
     <y>305</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;幼圆&quot;;</string>
   </property>
   <property name="text">
    <string>郑州</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_guangzhou">
   <property name="geometry">
    <rect>
     <x>452</x>
     <y>455</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;幼圆&quot;;</string>
   </property>
   <property name="text">
    <string>广州</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_xuzhou">
   <property name="geometry">
    <rect>
     <x>552</x>
     <y>255</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;幼圆&quot;;</string>
   </property>
   <property name="text">
    <string>徐州</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_shanghai">
   <property name="geometry">
    <rect>
     <x>592</x>
     <y>385</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;幼圆&quot;;</string>
   </property>
   <property name="text">
    <string>上海</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>448</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;幼圆&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>452</x>
     <y>195</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/res/bluePoint1.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>444</x>
     <y>290</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/res/bluePoint1.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>349</x>
     <y>284</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/res/bluePoint1.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>312</x>
     <y>384</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/res/bluePoint1.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>461</x>
     <y>427</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/res/bluePoint1.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>583</x>
     <y>366</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/res/bluePoint1.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>541</x>
     <y>277</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/res/bluePoint1.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>450</y>
     <width>92</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;幼圆&quot;;</string>
   </property>
   <property name="text">
    <string>查看详图</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
